apps/mobile/src/features/approver/utils.tsx

```diff
@@ -103,3 +110,49 @@ export function getStacksFeeData(feeType: FeeTypes) {
     time,
   };
 }
+
+interface ReviewTxSummaryProps {
+  tx: StacksTransactionWire;
+  symbol: CryptoCurrency;
+}
+export function formReviewTxSummary({ tx, symbol }: ReviewTxSummaryProps) {
```

Comment:
At first glance I thought this was a component. Maybe change the name to be `getFormReviewTxSummary`?

Also, how likely is it that the user can get here with a `SIP-10`? 🤔 Just wondering if we can prevent it sooner?